test/docker/integration/install/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:22.04 as install
```

Comment:
can we reuse the OSS docker files and add in multi-stages, one for the operating system and NGINX. The second for the agent. You can target specific stages in the docker file through the cli: see the options` --stage ` https://docs.docker.com/engine/reference/commandline/build/#options and this can be done in compose in a similar way: https://stackoverflow.com/a/53101932/515970